x-pack/plugins/apm/server/plugin.ts

```diff
@@ -103,28 +102,20 @@ export class APMPlugin
       });
     }
 
-    const ossTutorialProvider = plugins.apmOss.getRegisteredTutorialProvider();
-    plugins.home?.tutorials.unregisterTutorial(ossTutorialProvider);
-    plugins.home?.tutorials.registerTutorial(() => {
-      const ossPart = ossTutorialProvider({});
-      if (this.currentConfig!['xpack.apm.ui.enabled'] && ossPart.artifacts) {
-        // @ts-expect-error ossPart.artifacts.application is readonly
-        ossPart.artifacts.application = {
-          path: '/app/apm',
-          label: i18n.translate(
-            'xpack.apm.tutorial.specProvider.artifacts.application.label',
-            {
-              defaultMessage: 'Launch APM',
```

Comment:
What was this "Launch APM" button?

Reply:
Oh, I see it was just moved.

Reply:
This is no longer needed at this place, I'm adding it here https://github.com/elastic/kibana/pull/100780/files/7195da0ccd5e723b97894afa1fd7c9ec568bd0e8#diff-8b58ea65cacd0dd98b183ea7e4136f77912d0e81290e60ad2c24373c0282e5adR68-R79